jose/util/src/main/java/org/wildfly/security/jose/util/SystemPropertiesJsonParserFactory.java

```diff
@@ -97,7 +97,7 @@ public String getProperty(String key) {
                 return overrides.get(key);
             } else if (key.startsWith("env.")) {
                 return System.getenv().get(key.substring(4));
-            } else if (System.getProperties().contains(key)) {
+            } else if (System.getProperties().containsKey(key)) {
```

Comment:
No a reason to hold this PR but when is this called?  Just during deployment or from an app as well?

Just wondering if there is a security manager implication.

Reply:
This is called during servlet context initialization.

Reply:
Would that mean the deployments protection domain is on the call stack at that time?

Reply:
Just checked, this gets called just during deployment - the deployment's protection domain is not on the call stack at the time this is called.